### ISSUE TITLE
Update library types to fixed length types

### ISSUE THREAD

**Issue by arjo129** [arjo129/uSpeech#36]
With variations in various boards, it seems necessary to update the library to used `u_int16t` rather than `int`. (which was bad practice).

**Comment by q2dg** [comment]
Hello. Is someone out there?

**Comment by arjo129** [comment]
Hi, what would you like to know?

**Comment by q2dg** [comment]
Hi! If this issue has been resolved. Thanks

**Comment by arjo129** [comment]
No it hasn't been resolved.

**Comment by gonzalj3** [comment]
Hello!
I went ahead and made changes to the uspeech.h file so that it contains u_int16t types instead of int. I also included the appropriate library. I wanted to add a message here because this issue seems a bit dated and I am new to helping in open source projects. In other words I just wanted to confirm that I was helping in an existing issue and that I didnt break anything.

**Comment by arjo129** [comment]
Yep thanks I'll take a look at the PR just give me a day or two, bit busy. In future you can reference you're pull request by mentioning the pull request number like so `#PRnum` #44. Cheers!
